index.src.html

```diff
@@ -1477,40 +1465,14 @@
         adjustments.
```

Comment:
@sandersdan, for codedHeight and codedWidth, should we add notes to these that say how they're used (as a hint to guide selection of the [[codec impl]])?

Reply:
I think that's the only remaining use, yes. It might be more than a hint in that an implementation should be able to outright reject unsupported sizes.

Reply:
I added a note after the algorithm, but it might be better placed under the dict-member definitions.

The note itself might too ambiguous, should upper bounds checking be part of the algorithm?

Reply:
I removed the note following discussions with @chcunningham. This goes in the "unsupported" errors, and not the "invalid errors".

Reply:
I think this is addressed.